docs/doc/10-deploy/10-node-config/01-metasrv-config.md

```diff
@@ -2,113 +2,22 @@
-
-## 4. Raft config
-
-- `raft_config.id` is the globally unique id for this node; it is a `u64`.
-
-- `raft_config.raft_dir` is the local dir to store metadata, including raft log
-  and state machine etc.
-
-- `raft_config.raft_api_port`,`raft_config.raft_listen_host` and `raft_config.raft_advertise_host`
-  defines the service for internal raft communication.  Application should never touch this port.
-
-  `raft_listen_host` is the host the internal raft server listens on.
-  `raft_advertise_host` is the host the internal raft client to connect to.
-
-## 5. Raft internal config
-
-Defines raft behaviors on raft-storage and the state machine.
-
-- `heartbeat_interval` specifies the interval between two heartbeat in milliseconds.
-
-- `install_snapshot_timeout` specifies the max time for installing a snapshot in milliseconds.
-
-- `max_applied_log_to_keep` specifies the max number of applied raft-log to keep.
-
-- `snapshot_logs_since_last` specifies the number of raft-logs since the last snapshot beyond which a snapshot will be generated.
-
-- `wait_leader_timeout` specifies the max time for waiting a cluster leader in milliseconds.
-
-## 6. Startup config
-
-- `single` tells the node to initialize a single node cluster if it is not
-  initialized. Otherwise, this arg is just ignored.
-
-- `join` specifies a list of address(`<raft_advertise_host>:<raft_api_port>`) of nodes in an existing cluster a new node is joint to.
-
-  `join` is only used for an uninitialized node.
-  `join` will be ignored if the node is already initialized.
-
-## Configuring Meta Node with Environment Variables
\ No newline at end of file
+This page describes the Meta configurations available in the [databend-meta.toml](https://github.com/datafuselabs/databend/blob/main/scripts/distribution/configs/databend-meta.toml) configuration file.
+
+- Some parameters listed in the table may not be present in [databend-meta.toml](https://github.com/datafuselabs/databend/blob/main/scripts/distribution/configs/databend-meta.toml). If you require these parameters, you can manually add them to the file.
+
+- You can find [sample configuration files](https://github.com/datafuselabs/databend/tree/main/scripts/ci/deploy/config) on GitHub that set up Databend for various deployment environments. These files were created for internal testing ONLY. Please do NOT modify them for your own purposes. But if you have a similar deployment, it is a good idea to reference them when editing your own configuration files.
+
+| Parameter                       | Description                                                                                                             |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| log_dir                         | Directory where Databend stores its log files.                                                                          |
```

Comment:
`log_level` and `log_dir` are obsolete and should be removed from doc or marked as `deprecated`.

The replacement is `log: LogConfig`, which provide rich log config such as separate stderr log config and file log config. This doc should explain or add link to `log: LogConfig` here.